### PR TITLE
Update supported-platforms.md

### PR DESCRIPTION
**Pull request by khyati82**
Updating supported platforms based on meeting action items documented in meeting notes go/dash-platforms-support-governance

_Description of what this PR is changing or adding, and why:_

_Issues fixed by this PR (if any):_

## Presubmit checklist
- [ ] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [ ] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [ ] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/development/tools/sdk/release-notes/supported-platforms.md [hunk 1]
```diff
@@ -24,37 +40,22 @@ Flutter supports the following platforms:
...
+|Linux   |Ubuntu 18.04 LTS      |
+|macOS   |Monterey (12) & above |
 |Web     |Chrome 84             |
 |Web     |Firefox 72.0          |
 |Web     |Safari / Catalina     |
 |Web     |Edge 1.2.0            |
 |Windows |Windows 10            |
 
-* Passing tests on Android SDK 19 also confers a passing result on SDK 20.
+\* Passing tests on Android SDK 19 also confers a passing result on SDK 20.
```
Comment (domesticmouse) [inline src/development/tools/sdk/release-notes/supported-platforms.md:58]: Is this change intentional?

**Review by domesticmouse — APPROVED** [review]

File: src/development/tools/sdk/release-notes/supported-platforms.md (hunk 1 above)
Comment (khyati82) [reply src/development/tools/sdk/release-notes/supported-platforms.md:58]: Yes I am escaping the * character referenced above
